packages/api/src/controllers/stream.ts

```diff
@@ -401,6 +401,31 @@ app.get("/", authorizer({}), async (req, res) => {
   );
 });
 
+export async function getRecordingPlaybackUrl(
+  stream: DBStream,
+  objectStoreId: string
+) {
+  let url: string;
+
+  try {
+    const session = await db.stream.getLastSessionFromSessionsTable(stream.id);
+
+    if (!session) {
+      return null;
+    }
+
+    const os = await db.objectStore.get(objectStoreId);
+    url = pathJoin(os.publicUrl, session.playbackId, session.id, "output.m3u8");
+  } catch (e) {
+    console.log(`
+      Error getting recording playback url: ${e}
+    `);
+    return null;
+  }
```

Comment:
1. Shouldn't this have that same logic to try fetching the manifest then falling back to the secondary OS if not found? (we might wanna have a helper for that after all, it'd be repeated in 3 places now)
2. I don't think we should ignore errors. This should explode if we have problems accessing the DB. It's better to have the client retry than returning a false result here (as if the recording URL didnt' exist)

Reply:
1. implemented - as suggested by max!
2. I removed this try catch with that